snomed/com.b2international.snowowl.snomed.reasoner/src/com/b2international/snowowl/snomed/reasoner/request/ClassificationCreateRequest.java

```diff
@@ -101,8 +103,18 @@ public String execute(final BranchContext context) {
 				.addAllConcepts(additionalConcepts)
 				.build(branch.path());
 		
+		CodeSystem codeSystem = CodeSystemRequests.prepareSearchCodeSystem()
+				.build()
+				.execute(context)
+				.getItems()
+				.get(0);
```

Comment:
The currently accessed `TerminologyResource` should be available from the context; it is loaded early on during the execution of a request:

```suggestion
		CodeSystem codeSystem = (CodeSystem) context.service(TerminologyResource.class);
```